dependency.json

```diff
@@ -1,5 +1,9 @@
 {
     "include": {
-        "spryker/transfer": "Provides transfer objects definition with `::get*OrFail()` functionality."
+        "spryker/transfer": "Provides transfer objects definition with `::get*OrFail()` functionality.",
+        "spryker/testify": "False positive validation issue."
```

Comment:
it looks like dev dependency but it's added to main include

Reply:
Fixed.